ui/src/literature/components/CitationSummarySwitch.jsx

```diff
@@ -1,23 +1,44 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { BarChartOutlined } from '@ant-design/icons';
 import { Switch, Tooltip } from 'antd';
+import PropTypes from 'prop-types';
+import { WITH_CITATION_SUMMARY } from '../constants';
 
 const CHART_ICON = <BarChartOutlined />;
 
-function CitationSummarySwitch(props) {
-  const { checked } = props;
-
+function CitationSummarySwitch({
+  checked,
+  updateHash,
+  onChange,
+  isCitationSummaryEnabled,
```

Comment:
we should have a clear name to indicate this is user preferences, not the actually `enabled` state